operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ServiceAccountOperator.java

```diff
@@ -41,16 +42,15 @@ public ServiceAccountOperator(Vertx vertx, KubernetesClient client, boolean patc
     }
 
     @Override
-    protected Future<ReconcileResult<ServiceAccount>> internalPatch(String namespace, String name, ServiceAccount current, ServiceAccount desired) {
+    protected Future<ReconcileResult<ServiceAccount>> internalPatch(Reconciliation reconciliation, String namespace, String name, ServiceAccount current, ServiceAccount desired) {
         if (patching)   {
             if (desired.getSecrets() == null || desired.getSecrets().isEmpty())    {
                 desired.setSecrets(current.getSecrets());
             }
-
-            return super.internalPatch(namespace, name, current, desired);
+            return super.internalPatch(reconciliation, namespace, name, current, desired);
         } else {
             // Patching an SA causes new tokens to be created, which we should avoid
-            log.debug("{} {} in namespace {} has not been patched: patching service accounts generates new tokens which should be avoided.", resourceKind, name, namespace);
+            reconciliationLogger.debugCr(reconciliation, "{} {} in namespace {} has not been patched: patching service accounts generates new tokens which should be avoided.", resourceKind, name, namespace);
```

Comment:
Should this be named just LOGGER? It looks like this is wrong already in some parent class.